inc/namespace.php

```diff
@@ -34,6 +34,12 @@ function bootstrap() {
  * @return Group[] Sorted list of groupse
  */
 function get_documentation() : array {
+	static $documentation;
+
+	if ( $documentation ) {
```

Comment:
```suggestion
	if ( empty( $documentation ) ) {
```

Rather than the implicit cast to boolean (which I would like to warn about in the standards someday...)